static/app/components/alert.tsx

```diff
@@ -91,6 +91,7 @@ const Alert = styled(
     children,
     className,
     system: _system, // don't forward to `div`
+    css: _css,
```

Comment:
What's this for?

Reply:
I was getting tsc warnings because of it

Reply:
It wasn't broken before right?

Reply:
it was just a bit annoying but I can remove it from this PR 😉